andela/reactors-cards-for-humanity
# Chore/143412447/integrate coveralls

**What does this PR do?**

It involves integration of coverall and will update the package.json scripts adding coveralls command, the travis.yml file by adding the npm run coveralls command and the readme file by adding the badge.

**Description of Task to be completed?**

Integrating Coveralls to the project repo and adding a coveralls badge.

**How should this be manually tested?**

Edit any file then commit and make a push to repo.
Login into coveralls.io and locate your repo to see the coverage.

**What are the relevant pivotal tracker stories?**

https://www.pivotaltracker.com/story/show/143412447
